src/primitive/defs.rs

```diff
@@ -2956,9 +2956,8 @@ primitive!(
     /// ex: [1 5 2 9 11 0 7 12 8 3]
     ///   : ▽×⸮≥5:⸮≤10..
     (1, Trace, Debug, ("trace", '⸮'), Impure),
-    /// Debug print all the values currently on stack without popping them
+    /// Debug print all stack values after applying a function to them without popping them
```

Comment:
I prefer these short descriptions to be shorter, even if they aren't super specific.